images/hubble-proto/install-protoplugins.sh

```diff
@@ -14,11 +14,12 @@ go mod init github.com/cilium/hubble/protoc
 
 # latest tag at the time. For some reason `go get foo/bar/baz@vX.Y.Z` doesn't
 # work with nested go.mod definitions.
-go get google.golang.org/grpc/cmd/protoc-gen-go-grpc@cee815d
+# b2c5f4a == tag: cmd/protoc-gen-go-grpc/v1.0.0
+go get google.golang.org/grpc/cmd/protoc-gen-go-grpc@b2c5f4a
```

Comment:
Why are we not using `@1.0.0` here?

Reply:
The label is literally `cmd/protoc-gen-go-grpc/v1.0.0`, not `v1.0.0` (which would point to an older release of `grpc-go`, the most recent one being `1.32.0`). And this format does not work with `go get`:
```
$ go get google.golang.org/grpc/cmd/protoc-gen-go-grpc@cmd/protoc-gen-go-grpc/v1.0.0: google.golang.org/grpc/cmd/protoc-gen-go-grpc@cmd/protoc-gen-go-grpc/v1.0.0: invalid version: version "cmd/protoc-gen-go-grpc/v1.0.0" invalid: disallowed version string
```

Reply:
> The label is literally cmd/protoc-gen-go-grpc/v1.0.0

D: